### PR TITLE
Offert section2

### PR DESCRIPTION
**Pull request by agnieszkastanczuk**

### CODE REVIEW

File: src/components/Layout/Layout/index.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ const Layout: React.FC<Props> = ({
 
         {children}
       </div>
-      <Footer isContactPage={isContactPage} isWhiteBg={footerWhiteBg} />
+      {/* <Footer isContactPage={isContactPage} isWhiteBg={footerWhiteBg} /> */}
```
Comment (mrruby) [inline src/components/Layout/Layout/index.tsx]: Why Footer is hidden?

File: src/components/Offert/offert-full.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+              <img src="../../../assets/offert/photos.svg" alt="GetBold-photos" />
+            </div>
+            <div className="h-[84px] w-[84px] shadow-div rounded-small-item">
+              <img src="../../../assets/offert/video.svg" alt="GetBold-video" />
+            </div>
+            <div className="h-[84px] w-[84px] shadow-div rounded-small-item">
+              <img src="../../../assets/offert/all.svg" alt="GetBold-all" />
+            </div>
+          </div>
+        </div>
```
Comment (mrruby) [inline src/components/Offert/offert-full.tsx]: how about?

const items = [
  { src: "../../../assets/offert/social-media.svg", alt: "GetBold-socialmedia" },
  { src: "../../../assets/offert/photos.svg", alt: "GetBold-photos" },
  { src: "../../../assets/offert/video.svg", alt: "GetBold-video" },
  { src: "../../../assets/offert/all.svg", alt: "GetBold-all" },
];

[...]  
  
  <div className="flex gap-6">
    {items.map((item, index) => (
      <div key={index} className="h-[84px] w-[84px] shadow-div rounded-small-item">
        
      </div>
    ))}
  </div>

File: src/pages/oferta.tsx [hunk 3]
```diff
@@ -32,8 +36,12 @@ const OffertPage = () => {
           firstPartAdditionalParagraph="Zobacz, co konkretnie zrobimy, aby Twój biznes się "
           secPartAdditionalParagraph="rozkręcił!"
         />
-        <BoxesOffert />
-        <OffertComponent />
+        {/* <BoxesOffert /> */}
```
Comment (mrruby) [inline src/pages/oferta.tsx:39]: Why commented out?

**Review by mrruby — CHANGES_REQUESTED** [review]

File: src/components/Offert/offert-full.tsx (hunk 2 above)
Comment (mrruby) [reply src/components/Offert/offert-full.tsx]:
```
  <div className="flex gap-6">
    {items.map((item, index) => (
      <div key={index} className="h-[84px] w-[84px] shadow-div rounded-small-item">
        <img src={item.src} alt={item.alt} />
      </div>
    ))}
  </div>
  ```

File: src/components/Layout/Layout/index.tsx (hunk 1 above)
Comment (agnieszkastanczuk) [reply src/components/Layout/Layout/index.tsx]: I was just checking something, but the Footer should be visible.

File: src/pages/oferta.tsx (hunk 3 above)
Comment (agnieszkastanczuk) [reply src/pages/oferta.tsx:39]: BoxesOffert is outdated but this will be section 3, I haven't started doing it yet.

**Review by mrruby — APPROVED** [review]

**Pull request merged**